lib/engine/share_bundle.rb

```diff
@@ -49,5 +48,13 @@ def price
     def can_dump?(entity)
       !presidents_share || (corporation.share_holders.reject { |k, _| k == entity }.values.max || 0) > 10
     end
+
+    def to_bundle
+      self
+    end
+
+    def ==(other)
```

Comment:
where is this used?

Reply:
https://github.com/tobymao/18xx/blob/1846-bankruptcy/lib/engine/step/g_1846/train.rb#L91